.ci/packaging.groovy

```diff
@@ -245,19 +245,25 @@ def triggerE2ETests(String suite) {
 
   def branchName = isPR() ? "${env.CHANGE_TARGET}" : "${env.JOB_BASE_NAME}"
   def e2eTestsPipeline = "e2e-tests/e2e-testing-mbp/${branchName}"
+
+  def parameters = [
+    booleanParam(name: 'forceSkipGitChecks', value: true),
+    booleanParam(name: 'forceSkipPresubmit', value: true),
+    booleanParam(name: 'notifyOnGreenBuilds', value: !isPR())
+    string(name: 'runTestsSuites', value: suite),
+    string(name: 'GITHUB_CHECK_NAME', value: env.GITHUB_CHECK_E2E_TESTS_NAME),
+    string(name: 'GITHUB_CHECK_REPO', value: env.REPO),
+    string(name: 'GITHUB_CHECK_SHA1', value: env.GIT_BASE_COMMIT),
```

Comment:
I'm thinking that these three should be added only for PRs, thoughts?

Reply:
Uhm, if running those builds are PR agnostic, then the GH check will be reported even for branches and tags independently. I see value by having them, unless it's already triggered as part of another pipeline

Reply:
Ok then, let's maintain them, as they could help contributing the e2e tests check on a merge commit. Thank you!